src/components/functional/navBar.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import {
+  Collapse, Navbar, NavbarToggler, NavbarBrand, Nav, Button, Container,
+} from 'reactstrap';
+import Title from '../../assets/images/ah-logo-text.png';
+import Logo from '../../assets/images/ah_logo.png';
+
+class AppBar extends Component {
+  state = {
+    isToggled: false,
+  };
+
+  toggle = () => {
+    const { isToggled } = this.state;
+    this.setState({ isToggled: !isToggled });
+  }
+
+  render() {
+    const { isToggled } = this.state;
+    return (
+      <Navbar dark className="nav-top shadow-sm p-3" expand="sm">
+        <Container>
+          <NavbarBrand className="nav-logo">
```

Comment:
I would love this navbar logo to be clickable. When I click navbar, it takes me to `/` page, no matter the page I am. 
Make the whole logo image and text clickable and redirect always to homepage.